src/arrays/index.md

```diff
@@ -129,13 +128,13 @@ libaverages.a(std-9a66b6a343d52844.0.o): In function `std::sys::imp::mutex::{{im
 clang: error: linker command failed with exit code 1 (use -v to see invocation)
 ```
 
-Oops! When clang tried to compile our `libaverages.a` library and `main.c` into
-one executable it wasn't able to find a bunch of symbols.
+Oops! When clang tried to compile the libraries `libaverages.a` and `main.c`
+into one executable it wasn't able to find a bunch of symbols.
 
-Remember those notes from earlier? That's what `rustc` was trying to warn us
-about. When you compile everything statically you need to include **all** your
-dependencies. You didn't have this issue when dynamically linking because the
-loader finds everything for you.
+Remember those notes from earlier? That's what `rustc` was trying to warn
```

Comment:
Yup, so I can either,
1.  Add the notes link commit from #10, and close the pull request
2.  Wait until this PR is merged and then update #10 to be based off the updated master branch.